samples/msal-react-samples/b2c-sample/README.md

```diff
@@ -9,8 +9,6 @@ This sample was bootstrapped with [Create React App](https://github.com/facebook
 
 Implementing B2C user-flows is a matter of initiating authorization requests against the corresponding authorities. This sample demonstrates the [sign-up/sign-in](https://docs.microsoft.com/azure/active-directory-b2c/add-sign-up-and-sign-in-policy?pivots=b2c-user-flow) user-flow with [self-service password reset](https://docs.microsoft.com/azure/active-directory-b2c/add-password-reset-policy?pivots=b2c-user-flow#self-service-password-reset-recommended).
 
-> If you do need to support legacy user-flows, such as legacy password reset user-flow, please refer to the [Angular B2C sample](../../msal-angular-v2-samples/angular-b2c-sample-app/) for an illustration.
```

Comment:
I no longer think this is correct. Even the Angular v2 B2C sample was updated to use the self-service passport reset flow.